vendor/assets/stylesheets/susy/susy/language/susy/_box-sizing.scss

```diff
@@ -0,0 +1,47 @@
+// Susy Box Sizing
+// =================
+
+// Global Box Sizing
+// -----------------
+// Set a box model globally on all elements.
+// - [$box]: border-box | content-box
+// - [$inherit]: true | false
+@mixin global-box-sizing(
+  $box: susy-get(global-box-sizing),
+  $inherit: false
+) {
+  $inspect: $box;
+
+  @if $inherit {
+    @at-root {
+       html { @include output((box-sizing: $box)); }
+       *, *:before, *:after { box-sizing: inherit; }
```

Comment:
Line should be indented 6 spaces, but was indented 7 spaces<br>Each selector in a comma sequence should be on its own line